pcl_segment/positionRPY position
bool is_Known
